### DISCUSSION TITLE
Can't implement ocrit

### DISCUSSION THREAD

**Discussion by MIIO73** [insidegui/ocrit#3] [Q&A] [answered]
Hi,

I'am a newbie and I have a question about your command line tool OCRIT. The file works very well, but I have no idea how can I set the language parameter.

Which language code is the right for Germany? And how should the command line look like?

ocrit  path/image.jpg --output path --language germany ????

Can you please help to run OCRIT with the right language 

 PLEASE :-)

**Comment by insidegui** [comment, marked as answer]
For German, you'd use the language code `de-DE`, like so:

`ocrit image.png -l de-DE`

If you run the command with an unsupported language code, it will list the supported language codes (which vary according to the version of macOS you're on):

```
$ ocrit image.png -l lalala
Validating images…
Performing OCR with language: lalala…
OCR failed for image.png: Unsupported language "lalala". Supported languages are en-US, fr-FR, it-IT, de-DE, es-ES, pt-BR, zh-Hans, zh-Hant
```
